lib/y-richtext.coffee

```diff
@@ -119,14 +128,40 @@ class YRichText extends BaseClass
   # @param position [Integer] the position where to insert it
   setCursor : (position) ->
     @selfCursor = @_model.getContent("characters").ref(position)
+
     @_model.getContent("cursors").val(@_model.HB.getUserId(), @selfCursor)
 
+  setAuthor : (option) ->
```

Comment:
From the maintainers perspective, I don't think it makes sense to implement colors, and mappings from userid->color&name in this module. y-richtext is about richtext, nothing more. It should be so general, that it only handles the rich text structure, and transformations to ot operations. Everything else should be outsourced. You can leave it in y-richtext for now, but put everything related to colors & mappings in one dedicated module.